client/settings/phone-input/index.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState, useRef } from 'react';
+import { __ } from '@wordpress/i18n';
+import intlTelInput from 'intl-tel-input';
+import './style.scss';
+
+/**
+ * Internal dependencies
+ */
+import utils from 'iti/utils';
+
+interface PhoneNumberInputProps {
+	value: string;
+	onValidationChange: ( isValid: boolean ) => void;
+	onValueChange: ( value: string ) => void;
+	inputProps: {
+		label: string;
+		ariaLabel: string;
+		name: string;
+	};
+}
+
+const PhoneNumberInput = ( {
+	onValueChange,
+	value,
+	onValidationChange = ( validation ) => validation,
+	inputProps = {
+		label: '',
+		ariaLabel: '',
+		name: '',
+	},
+}: PhoneNumberInputProps ): JSX.Element => {
+	const [
+		inputInstance,
+		setInputInstance,
+	] = useState< intlTelInput.Plugin | null >( null );
+	const inputRef = useRef< HTMLInputElement >( null );
+
+	const handlePhoneNumberInputChange = () => {
+		if ( inputInstance ) {
+			onValueChange( inputInstance.getNumber() );
+			onValidationChange( inputInstance.isValidNumber() );
+		}
+	};
+
+	const removeInternationalPrefix = ( phone: string ) => {
+		if ( inputInstance ) {
+			return phone.replace(
+				'+' + inputInstance.getSelectedCountryData().dialCode,
+				''
+			);
+		}
+
+		return phone;
+	};
+
+	useEffect( () => {
+		let iti: intlTelInput.Plugin | null = null;
+		const currentRef = inputRef.current;
+
+		const handleCountryChange = () => {
+			if ( iti ) {
+				onValueChange( iti.getNumber() );
+				onValidationChange( iti.isValidNumber() );
+			}
+		};
+
+		if ( currentRef ) {
+			iti = intlTelInput( currentRef, {
+				initialCountry: 'US',
+				customPlaceholder: () => '',
+				separateDialCode: true,
+				hiddenInput: 'full',
+				utilsScript: utils,
+			} );
+			setInputInstance( iti );
+			onValidationChange( iti.isValidNumber() );
+
+			// Focus the phone number input when the component loads.
+			currentRef.focus();
```

Comment:
In the Card Readers page, when the form loads  the input phone component gets the focus. It feels a bit weird since the phone input is not the first in the form. Not sure how hard it can be to handle it in a different way, without giving the focus to the component?

Reply:
Oh yes, I agree. This was part of [another review](https://github.com/Automattic/woocommerce-payments/pull/4355#discussion_r903421420) but I missed it. Thank you for pointing it out!